index.src.html

```diff
@@ -91,11 +92,10 @@
       text: creation URL
     urlPrefix: workers.html
       text: running a worker; url: run a worker
-    urlPrefix: dom.html
-      text: document referrer policy; url: concept-document-referrer-policy; for: Document
   type: interface
     urlPrefix: dom.html
       text: Document
+      text: referrer policy; url: concept-document-referrer-policy; for: Document
```

Comment:
No, that's not right; interfaces are Web IDL things declared as `interface X`. I'm surprised Bikeshed even lets you use an "interface" with a space in the name...